luxonis_ml/data/parsers/coco_img_with_ann_parser.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple
+
+from .base_parser import BaseParser
+from .coco_parser import COCOParser
+
+
+class COCOImgWithAnnParser(COCOParser):
+    """Parses directory with COCO annotations to LDF. Unlike COCOParser here images and
```

Comment:
```suggestion
    """Parses directory with COCO annotations to LDF. Unlike COCOParser, images and
```

Little nitpick for read-ability